02Lesson/01Homework/index.html

```diff
@@ -1,8 +1,54 @@
+            <h6> Narcos </h6>
+
+      <h3>Favorite Websites
+      </h3>
+      <ul>
+
+        <li><a href="www.bleacherreport.com">Bleacher Report</a></li>
```

Comment:
Those lines might get long when adding styles. Better put the block elements `li` on their own line and indent child elements.